share/doc/wake/basic_wake_tutorial.md

```diff
@@ -0,0 +1,660 @@
+## Wake Tutorial
+
+### Invoking wake
+
+1. Create a wake database: `wake --init .`
```

Comment:
I am loving this tutorial. First off, can we have some definition... 

what is the wake database ? 
What will be in the database ? 
Once I do wake --init . in certain directory, when do I need to do it again ?

Reply:
You can delete the wake.db file and again do `wake --init .` and restart it from the scratch.